README.Rmd

```diff
@@ -95,15 +95,15 @@ install.packages("shiny.semantic")
 and the latest version with devtools:
 
 ```{r, eval=FALSE}
-devtools::install_github("Appsilon/`r params$repo_name`")
+devtools::install_github("Appsilon/shiny.semantic@develop")
```

Comment:
Let's use `remotes::install_github`. Rstudio tends to use that package now as it's way more lightweight.